Alameen688/MyDiary
# 159483112 Frontend Client should be able to see list of entries and also view a particular entry from the list

#### What does this PR do?
Implements a working frontend list entries and view entry by connecting ui template and server api
endpoint for get all entries and get entry by id. A change to the
index.html is also included to remove unnecessary footer text (based on LFA feedback)
#### Description of Task to be completed?
The following should be working
- List all diary entry for logged in user
- View a particular by id for a logged in user
#### How should this be manually tested?
After cloning the repo, cd into it then
- make sure you have postgresql installed on your machine and start it.
- add a postgreSQL DATABASE_URL to your .env file
   ```
     DATABASE_URL=postgresql://username:password@localhost:5432/mydiarydb
   ```
 - RUN  `npm run dev` to start the server
 - Navigate to `/client/list-entry.html`, you should be able to view a list of all entries for the logged in user
 - Navigate to the address of one of the entries on the list `/client/view-entry.html?id=<entry_id>`, you should be see the details for that particular entry with the specified <entry_id>
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
[#159483112](https://www.pivotaltracker.com/story/show/159483112)
